src/pages/SendTokens/InfoCircle.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { Typography } from '@mui/material';
+
+
+const InnerCircle = styled('div')({
+  width: '9rem',
+  height: '9rem',
+  borderRadius: '50%',
+  border: '4px solid rgb(134, 194, 50)',
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+});
+
+const InnerText = styled(Typography)({
+  fontSize: 24,
+  lineHeight: 1,
+  fontWeight: 700,
+  color: 'rgb(134, 194, 50)',
+});
+
+const InnerSubtext = styled(Typography)({
+  fontSize: 12,
+  lineHeight: '16px',
+  fontWeight: 400,
+  marginTop: '1em',
+  color: 'rgb(134, 194, 50)',
+});
+
+//TODO refactor this and WalletInfoBlock
+const InfoCircle = (props) => {
```

Comment:
If you want to create any refactoring, you should better create an issue in Git and describe which exactly should be refactored instead of adding comments here.

Reply:
Should I reverse the changes I made to `MenuItem.js` then?

Reply:
No, it's fine